File: blackWhite.swift
import Foundation

let input = readLine()!.split(separator: "")
let A = String(input[0])
let B = String(input[1])

let lastDigitA = Int(String(A.last!))!
let lastDigitB = Int(String(B.last!))!

if(lastDigitA+lastDigitB%2==0){
    print("Black")
}else{
    print("White")
}

File: blackWhite.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
string A,B;
cin>>A>>B;

int lastDigitA = A.back() - '0';
int lastDigitB = B.back() - '0';

if((lastDigitA+lastDigitB)%2==0)
cout<<"Black"<<endl;
else
cout<<"White"<<endl;

    return 0;
}